testutil.go

```diff
@@ -53,9 +64,86 @@ func (n *TestNetwork) AddNode(t testing.TB, startingBalance uint64) *TestLedger
 	return node
 }
 
+func (n *TestNetwork) Nodes() []*TestLedger {
+	return append(n.nodes, n.faucet)
+}
+
+// WaitForRound waits for all the nodes in the network to
+// reach the specified round.
+func (n *TestNetwork) WaitForRound(t testing.TB, round uint64) {
+	if len(n.nodes) == 0 {
+		return
+	}
+
+	done := make(chan struct{})
+	go func() {
+		for _, node := range n.nodes {
+			for ri := range node.WaitForRound(round) {
+				if ri == round {
+					break
+				}
+			}
+		}
+
+		close(done)
+	}()
+
+	select {
+	case <-time.After(time.Second * 10):
+		t.Fatal("timed out waiting for round")
+
+	case <-done:
+		return
+	}
+}
+
 func (n *TestNetwork) WaitForConsensus(t testing.TB) {
-	all := append(n.nodes, n.faucet)
-	TestWaitForConsensus(t, time.Second*30, all)
```

Comment:
that was the only usage of `TestWaitForConsensus`, so its not used anywhere either now